diffsims/sims/diffraction_simulation.py

```diff
@@ -200,10 +203,13 @@ def get_plot(self, g_max, annotate_peaks=True, with_labels=True, fontsize=12):
         fontsize : integer
             Fontsize for peak labels.
         """
+
+        label_hkl = get_unique_families(hkls.keys())
```

Comment:
at a glance, I think changing `hkls.keys()` to `self.hkls.keys()` might get you going? `hkls` is an attribute of a `ProfileSimulation`, which in this layout is "`self`"

Reply:
Now it looks like it should probably just be `self.hkls` because that's a list and not a dictionary?